### PR TITLE
Added more props to patient_attachments guide request

### PR DESCRIPTION
**Pull request by hagen**
Added a full request to the patient attachments upload guide, demo'ing how to set description and filename props.

### CODE REVIEW

File: guides/uploading_patient_attachments.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ To do so, make a POST request to the patient attachments endpoint, with the pati
 `https://api.cliniko.com/v1/patient_attachments`
 ```json
 {
-  "patient_id": "382",
+  "description": "Custom description for this attachment",
+  "filename": "The name of the file.txt",
```
Comment (JohnColvin) [inline guides/uploading_patient_attachments.md]: I don't think we allow setting this do we?

Comment (rclarke1) [reply guides/uploading_patient_attachments.md]: I have been setting description via this method, which is a very useful function.

Comment (hagen) [reply guides/uploading_patient_attachments.md]: Gah - will remove `filename`

**Comment by rclarke1** [comment]
What is the character limit of the the description field? I have occasionaly been getting a 422 Unprocessable Entity error, which I think might relate to the description length being too long.

**Comment by rclarke1** [comment]
> What is the character limit of the the description field? I have occasionaly been getting a 422 Unprocessable Entity error, which I think might relate to the description length being too long.

Actually I see from the front end that it is 255 characters

**Pull request merged**